docs/reference/verify-assets.mdx

```diff
@@ -106,11 +109,68 @@ As the SBOM was signed with our private key, you can now inspect it to track dow
 
 ## Verify image-based gadgets
 
-### With `ig`
-
 Like our container image, we sign all our image-based gadgets.
-The signature are verified by default using Inspektor Gadget public key:
+The signature are verified by default using Inspektor Gadget public key.
+
+<Tabs groupId="env">
+<TabItem value="kubectl-gadget" label="kubectl gadget">
+Compared to `ig`, you cannot specify information with regard to verification when calling `kubectl gadget run`:
+
+```bash
+$ kubectl gadget run --public-keys="$(cat your-key.pub)" trace_exec
+Error: unknown flag: --public-keys
+$ kubectl gadget run --verify-image=false trace_exec
+Error: unknown flag: --verify-image
+```
```

Comment:
I still think it would suffice to explain that in a paragraph above rather than having another terminal box that invites you to try it out and remember, but okay.